libdnf/utils/locker.cpp

```diff
@@ -29,7 +31,7 @@ namespace libdnf::utils {
 Locker::LockResult Locker::lock() {
     lock_fd = open(path.c_str(), O_CREAT | O_RDWR, 0660);
     if (lock_fd == -1) {
-        return LockResult::ERROR_FD;
+        throw SystemError(errno, "Failed to open lock file {}", path);
```

Comment:
I've been trying to consistently wrap strings in error messages in quotes, could you add them to your messages too?
```c++
throw SystemError(errno, "Failed to open lock file \"{}\"", path);
```